src/platform/mhv/downtime/components/MHVDowntimeApproaching.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatDatetime, formatElapsedHours } from '../utils/date';
+
+function MHVDowntimeApproaching({
```

Comment:
There is a similar code here. Not sure if it can be refactored cleanly. Maybe passing a flag?

Reply:
That thought had crossed my mind at one point, but given the back and forth on what the content would be for each alert component, and whether one alert would be dismissible, it seemed like a premature optimization.

Also sorry for not responding to this comment… Github was apparently hiding this even when all the other comments were resolved